src/prop/model.py

```diff
@@ -136,6 +137,8 @@ def PropagationLoss(self, f, lat1, lng1, h1, lat2, lng2, h2, land_cat=''):
       
     if land_cat == 'RURAL' or h1 >= 200 or h2 >= 200:
       itm_loss = self.ITM_AdjustedPropagationLoss(lat1, lng1, h1, lat2, lng2, h2, f, 0.5)
+      if land_cat == 'RURAL':
```

Comment:
We don't want losses to be random -- that means different applications will get different answers. We set vegetative loss to 0 in prop TG, as with other clutter losses.